QUICKSTART.md

```diff
@@ -0,0 +1,23 @@
+# Local Quickstart
```

Comment:
Flag that it would be good to have this in Spanish too. If we can't fit that in here, let's make an issue for it.

Reply:
How is this normally done @switzersc-usds?

Reply:
We go through our typical State Department translation process or someone on the team with Spanish chops scrubs in. cc @esfoobar-usds 

(We can also make an issue for this if we don't have time to get this in before end of sprint, in the interest of keeping issues self-contained within the sprint)

Reply:
I also like the idea of a follow up issue for this, since this seems like a different kind of work.